### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by HerrMuellerluedenscheid** [gpanders/meta-scipy#1]
Hi,
First of all, thanks for that recipe!! I'm rather new to yocto and was struggling to include scipy into my build. I create a dedicated layer for your scipy recipes because I failed to compile it simply by copying the content of this repo into the meta-openembedded subdirectories. I received errors of an invalid RPATH which I think was meant to be solved by patching numpy as you describe in the README.
But most likely I did something wrong copying this recipe into meta-oe. Could you maybe add one or two sentences on how I can add this recipe do my build?
Thanks!

**Comment by HerrMuellerluedenscheid** [comment]
Hi again,
Maybe to extend this issue a little: It would be great to have this recipe merged into the actual openembedded layer given that scipy is a pretty important dependency. I have no experience on what QC there needs to be done beforehand. Would you still consider this an option?

**Comment by gpanders** [comment]
I added some simple installation instructions in 6cb9636. Let me know if that clarifies anything.

If you run into further issues, I'll happily address any problems that are relevant to this recipe.

**Comment by HerrMuellerluedenscheid** [comment]
Makes it crystal clear! Thanks :+1:

**Comment by HerrMuellerluedenscheid** [comment]
P.s.: Added some advertisement here: https://stackoverflow.com/a/60524349/3437167
